environment_setup/install_openocd.sh

```diff
@@ -32,4 +32,6 @@ sudo make install
 
 echo "================================================================"
 echo "Done Installing OpenOCD"
+echo ""
+echo "Make sure to logout or restart your computer for permissions changes to take effect!"
```

Comment:
should we do `sudo systemctl restart udev` or is that too invasive? I think that would avoid having to restart

Reply:
Oh I didn't know you could do that. @garethellis0 @EvanMorcom @jonathanlew thoughts?

Reply:
I'm fine with that, let's just make sure to test it locally first.